eng/common/scripts/update-docs-ci.ps1

```diff
@@ -1,70 +0,0 @@
-#Requires -Version 6.0
```

Comment:
Isn't this script still useful but in our other automation process or will we need to rewrite it completely?

Reply:
I did not realize we have any other automation depends on this. We can leave it here, but remove the step only.

Reply:
I'm not sure if we are using it elsewhere that was a question for you.

Reply:
For my knowledge, it is only used for update target CI which is docs repo update. So once we want to remove the step, the script is not in use anymore.

Reply:
Sounds good. I just didn't want to break anything and I also didn't want to remove it if you planned to add it back for new implementation we plan to run in the docindex pipelines.